examples/games/crystal_ball/shaders/ground.frag

```diff
@@ -0,0 +1,55 @@
+#version 460 core
+
+precision highp float;
+
+#include <flutter/runtime_effect.glsl>
+
+uniform vec2 uSize;
+uniform float uPixelSize;
+uniform float uWaterLevel;
+uniform float uTime;
+
+uniform sampler2D tGameCanvas;
+
+out vec4 fragColor;
+
+vec4 fragment(vec2 uv) {
+    vec4 waterColor = vec4(1.0);
+    vec2 reflectedUv = uv.xy;
+
+    float fogIntensity = 0.0;
+    vec4 fogColor = vec4(1, 0.6, 1.0, 0.2);
+
+    if(uv.y >= uWaterLevel) {
```

Comment:
is this standard formatting? I typically prefer `if (`, etc (lots of places)

Reply:
My local formatter formatted as such.